spec/components/schemas/Acl.yaml

```
@@ -0,0 +1,15 @@
+type: array
+items:
+  type: object
+  required:
+    - scope
+    - permissions
+  properties:
+    scope:
```

```suggestion
    scopes:
```
Shouldn't it be `scopes`, because it's an array of scopes?